README.md

```diff
@@ -87,6 +87,38 @@ Run this to format non-test scala files: `sbt scalafmt`
 
 Run this to format test scala files: `sbt test:scalafmt`
 
+### SBT Alias Commands
+
+To run all unit and IT tests, check code coverage, and generate a report: Use `sbt testAndCoverage`
+
+Before raising a PR, to run all unit and IT tests, format the code, check code coverage, and generate a report: Use `sbt prePR`
+
+Before merging, to run all unit and IT tests, verify code formatting, check code coverage, and generate a report: Use `sbt preMerge`
+
+### Testing
```

Comment:
I think you have some duplication in this file - please double check and where possible make it match the layout and order of the frontend read me

Reply:
:facepalm: not sure how I've managed that